Added an inclusion of, "This setting is off by default..." (#42501)
elastic/kibana: docs/management/advanced-options.asciidoc

be inconsistent because different shards might be in different refresh states.
`search:includeFrozen`:: Includes {ref}/frozen-indices.html[frozen indices] in results. 
Searching through frozen indices

> might increase the search time. This setting is off by default. Users will have to opt-in to include frozen indices in results.

Reviewer: ```suggestion
might increase the search time. This setting is off by default. Users must opt-in to include frozen indices.
```
